android/java/org/chromium/chrome/browser/BraveHelper.java

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
+
+import org.chromium.base.CommandLine;
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.browser.ChromeSwitches;
+import org.chromium.chrome.browser.datareduction.DataReductionPromoUtils;
```

Comment:
Nit: Didn't seem like this import is being used, unless I'm just overlooking it.

Reply:
yep, I missed it, I wanted first to call a function directly from that class, but it does calls to native which isn't initialised on that phase yet.